index.md

```diff
@@ -13,107 +12,166 @@ First we set up & capture the content, then we render it in the scaffolding belo
-[![Network screenshot]({{ site.baseurl }}/images/site/screenshot-network.png)]({{ site.baseurl }}/images/site/screenshot-network.png){:.screenshot.zoom}
-### No interference
+### Packages included
+[Installing and running Cockpit]({{ site.baseurl }}/running.html) is simple. It's already included in most of the major distributions.
+
+---
+
+## Integrated
+
+### Plays well with others
 Jumping between the terminal and the web tool is no problem. A service started via Cockpit can be stopped via the terminal. Likewise, if an error occurs in the terminal, it can be seen in the Cockpit journal interface.
-{% endcapture %}
 
+[![Network screenshot]({{ site.baseurl }}/images/site/screenshot-network.png)]({{ site.baseurl }}/images/site/screenshot-network.png)
+{:.screenshot.zoom}
+
+### Sign in like normal
+By default, Cockpit uses [system credentials]({{ site.baseurl }}/guide/latest/privileges). You don't need to set up any special accounts. Network-wide logins are also supported through [single-sign-on]({{ site.baseurl }}/guide/latest/sso) and other [authentication]({{ site.baseurl }}/guide/latest/authentication) techniques.
+
+### Self-contained
+Cockpit is self-contained and uses existing system resources. You don't have to worry about setting up a webserver just to use Cockpit.
+
+### Uses existing system APIs
+Cockpit uses APIs that already exist on the system. It doesn't reinvent subsystems or add a layer of its own tooling.
+
+---
+
+## Get an overview
+
+### Understand at a glance
+Immediately understand the health of your server. Cockpit's overview page shows current statistics and the status of your system.
 
-{% capture blurb_multiserver %}
-[![Dashboard screenshot]({{ site.baseurl }}/images/site/screenshot-dashboard.png)]({{ site.baseurl }}/images/site/screenshot-dashboard.png){:.screenshot.zoom}
 ### Multi-server
-You can monitor and administer several servers at the same time. Just add it easily and your server will look after its buddies.
+You can monitor and administer [several servers]({{ site.baseurl }}/guide/latest/feature-machines.html) at the same time. Just add it easily and your server will look after its buddies.
+
+[![Dashboard screenshot]({{ site.baseurl }}/images/site/screenshot-dashboard.png)]({{ site.baseurl }}/images/site/screenshot-dashboard.png)
+{:.screenshot.zoom}
+
+### Troubleshoot
+Fix pesky problems with ease. No longer worry about SELinux security violations — the fix is just a click away.
+
+---
+
+## Open-ended
+
+### Pocket-sized
+Around the office or on the road? Use your phone's browser to check and manage systems.
+
+### Learn with Cockpit
+Not sure what you can do with your servers? Cockpit's friendly interface lets you explore and use features you might not even realized existed!
+
+### Extendable
+Have custom tasks? You can [write your own modules]({{ site.baseurl }}/blog/cockpit-starter-kit.html) to plug into Cockpit. We're also working on making this even easier in the near future.
```

Comment:
I would cut the part where it makes promises about the future. I think this page should reflect what it does now.